samples/msal-browser-samples/HybridSample/README.md

```diff
@@ -0,0 +1,198 @@
+});
+```
+
+Next, parse the authorization code, and invoke the `acquireTokenByCode` API on the `ConfidentialClientApplication` instance. 
+
+When invoking this API, set `returnSpaCode` to `true`, which will enable MSAL to acquire a second authorization code to be redeemed by your single-page application. 
+
+Your application should parse this second authorization code, as well as any account hints (e.g. `sid`, `login_hint`, `preferred_username`) and return them such that they can be rendered client-side:
+
+Source: [routes/auth.js](./routes/auth.js)
+```typescript
+router.post('/server-redirect', (req, res) => {
+    const tokenRequest = {
+        code: req.body.code,
+        scopes: ["user.read"],
+        redirectUri: "http://localhost:3000/auth/server-redirect",
+        returnSpaCode: true
+    };
+
+    msalInstance.acquireTokenByCode(tokenRequest)
+        .then((response) => {
+            const {
+                sid, // Session ID claim, used for non-hybrid
+                login_hint: loginHint, // New login_hint claim (used instead of sid or email)
+                preferred_username: preferredUsername // Email
+            } = response.idTokenClaims;
+
+            // Spa auth code that will be redeemed by MSAL.js v2 client-side
+            const { code } = response;
+
+            // Attach auth artifacts to session to they can be rendered downstream
+            req.session.isAuthenticated = true;
+            req.session.code = code;
+            req.session.sid = sid;
+            req.session.loginHint = loginHint;
+            req.session.preferredUsername = preferredUsername;
+
+            res.redirect(`/auth/client-redirect`)
+        })
+        .catch((error) => {
+            console.timeEnd(timeLabel)
+            console.log(error);
+            res.status(500).send(error);
+        });
+});
+```
+
+
+### Public client
+
+First, configure a new `PublicClientApplication` from MSAL.js in your single-page application:
+
+Source: [views/client-redirect.hbs](./views/client-redirect.hbs)
+```typescript
+const msalInstance = new msal.PublicClientApplication({
+    auth: {
+        clientId: "{{clientId}}",
+        redirectUri: "http://localhost:3000/auth/client-redirect",
+        authority: "https://login.microsoftonline.com/common"
+    }
+})
+```
+
+Next, render the `code` that was acquired server-side, and provide it to the `acquireTokenByCode` API on the MSAL.js `PublicClientApplication` instance. Be sure to not include any additional scopes that were not included in the first login request, otherwise the user may be prompted for consent.
+
+The application should also render any account hints, as they will be needed for any interactive requests to ensure the same user is used for both requests.
+
+Source: [views/client-redirect.hbs](./views/client-redirect.hbs)
+```typescript
+const sid = "{{sid}}";
+const code = "{{code}}";
+const preferredUsername = "{{preferredUsername}}";
```

Comment:
`sid` and `preferredUsername` aren't used below. Might create confusion.

Reply:
Good catch.